lib/search/helpArticleSearch.ts

```
@@ -0,0 +1,41 @@
+import Fuse from 'fuse.js';
+
+export type HelpArticle = {
+  title: string;
+  url: string;
+};
+
+export type SearchResult = HelpArticle;
+
+export function searchHelpArticles(articles: HelpArticle[], query: string, limit = 10): SearchResult[] {
+  const trimmedQuery = query.trim();
+  
```

ESLint

[error] Delete `··`

(prettier/prettier)